### ISSUE TITLE
Schema syntax highlighting doesn't work when extension something other than ".gql"

### ISSUE THREAD

**Issue by aneilbaboo** [kumarharsh/graphql-for-vscode#58]
(Moved to this repo from https://github.com/Mayank1791989/gql/issues/77)

How can I get syntax highlighting for schema files with extensions other than ".gql"?

For example, https://github.com/sysgears/apollo-universal-starter-kit has schema files named `schema.graphqls`.

I expected the following .gqlconfig file to work. The file is recognized (I get documentation popovers, which I'm unable to screencap, but trust me, they're there), however, the syntax isn't highlighted.  If I change my gqlconfig schema files to '**/schema.gql', rename the extension to .gql and reload, I see syntax highlighting.

```javascript
// .gqlconfig (with query)
{
  schema: {
    files: '**/schema.graphqls'
  },
  query: {
    files: [
      // query gql files
      {
        match: 'src/**/*.gql',
        parser: 'QueryParser',
      },
      // [Embedded queries] relay files
      {
        match: { include: 'src/**/*.js', ignore: '**/*.spec.js' },
        parser: [ 'EmbeddedQueryParser', { startTag: 'Relay\\.QL`', endTag: '`' } ],
        isRelay: true,
      },
      // [Embedded queries] gql tag files
      {
        match: { include: 'src/**/*.js', ignore: '**/*.spec.js' },
        parser: [ 'EmbeddedQueryParser', { startTag: 'gql`', endTag: '`' } ],
      }
    ]
  }
}
```

If would be great if syntax highlighting happened automatically for files listed in .gqlconfig glob patterns.

**Comment by kumarharsh** [comment]
The syntax highlighting is defined via the `grammars` key in the package.json file of the plugin - this is a static definition and thus cannot be loaded dynamically by reading your .gqlconfig.

But turning on syntax-highlighting for your own graphql files is very simple:
1. Open the file you want to turn on syntax highlighting for.
2. <kbd>Ctrl+P</kbd> > **Change Language Mode** > Select **GraphQL**
3. If you want to make this association permanent so that vscode always treats `.graphqls` files as graphql files, then follow these steps:
<kbd>Ctrl+P</kbd> > **Change Language Mode** > Select **Configure File Association for '.graphqls'** > Select **GraphQL**.

**Comment by kumarharsh** [comment]
Closed by 4a7568c
